data/hails.txt

```diff
@@ -700,9 +747,16 @@ phrase "friendly civilian"
 		"rotten fruit"
 		"sludge"
 		"unprocessed sewage"
+		"protest signs"
+		"water"
+		"robots"
+		"holy symbols"
+		"holy books"
+		"memes"
```

Comment:
Umm, these don't really fit at all.

`"Just made a big delivery of [protest signs || water || robots || holy books || memes], and since my poor ship's cargo hold smells awful!"`

Reply:
Memes would at least make them stink. Some holy books are stinky, subjectively. Maybe the robots are sewer robots. Protestors smell.

Reply:
saying holy stuff does may offend people, memes aren't really physical, robots depend; they're not naturally something that smells bad, and protestors are just humans (if I understand what ya meant by that)

Reply:
You took me too seriously, I was removing it but what I did was an attempt at humor. I'm not funny. Fixed!~

Reply:
@greatkek, these suggestions do not fit at all within the tone of the game. You are not a memelord and this is not 4chan or Reddit. We are trying to make a serious game here.

Reply:
Since we have openly religious hails, it might make sense to have openly atheist hails as well. But I think that they should be sincere and not memes. Stuff like "I hope you aren't another Jesus/Buddha/Yahweh/Tao worshiper. What a bunch of fools!"